addons/web/static/src/views/list/list_controller.js

```diff
@@ -434,9 +434,9 @@ ListController.props = {
     archInfo: Object,
 };
 ListController.defaultProps = {
+    canSelect: true,
     createRecord: () => {},
     editable: true,
-    hasSelectors: true,
     selectRecord: () => {},
```

Comment:
mouarf, we have `canSelect` and `selectRecord`. It looks like they're related, but they aren't at all.